Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7
+FROM python:3.7.6
```

Comment:
Is python 3.7.6 *required* or is python:3 sufficient?

Reply:
@ckolos, I gave some thought to this question and talked with few other Python developers and my decision is to keep it as it is: pin to the latest `3.7` patch version. Here are my reasons:

- Shavar does not regularly deployed. As a result pinning at `3`  or `3.7` can result into the service running in the same version of python without the maintainer being alerted via Dependabot about the service running in the older version of Python.
- I worry that automatically running the latest without personally testing the service can lead to unexpected and large breakages--especially since some parts of the unit-tests are mocking out the functionality of a package the service is using (e.g. moto to mock out boto requests).